subjects/blockchain/financial-instruments/README.md

```diff
@@ -131,52 +131,48 @@ Total available</p>
   </tr>
 </table>
 
-A visualization part allows the user to have a sense of his possession in TRG. For instance, a pie chart proportional to the current price of each asset can be displayed. For instance, assuming the default prices, the table above should render this way. 
+A visualization part allows the user to have a sense of his possession in TRG. For instance, a pie chart proportional to the current price of each asset can be displayed. For instance, assuming the default prices, the table above should render this way.
 
-  ![alt_text](viz.png "Vizualisation")
+![alt_text](viz.png 'Vizualisation')
 
 ### FAQ page
-An FAQ page that explains how to use the platform 
 
+An FAQ page that explains how to use the platform
 
-## The server
+#### The server
 
 The server serves the fronted using the database information, exposes an API for the frontend and interacts with the blockchain. Its API must offer a function to monitor deposits, triggered from the user interface, that verifies on the blockchain the transaction that sent the funds and another one to authorise withdrawals.
 
 The database contains
-* A table with the assets usable on the platform, their type, the address of their smart contract, and their price history.
-* A table of the users of the platform registered after the first connection. They must provide their legal name and upload a “passport” picture. No check is made when login in, but the picture is stored for future reference
-* A table of all ongoing sell and buy offers on the platform
 
+- A table with the assets usable on the platform, their type, the address of their smart contract, and their price history.
+- A table of the users of the platform registered after the first connection. They must provide their legal name and upload a “passport” picture. No check is made when login in, but the picture is stored for future reference
+- A table of all ongoing sell and buy offers on the platform
 
-## The vault smart contract
+#### The vault smart contract
```

Comment:
Does it need to be 4 hashes or 3 like the others?

Reply:
done